diff --git a/node_modules/resume-cli/build/render-html.js b/node_modules/resume-cli/build/render-html.js
index febd7dd..e738c88 100644
--- a/node_modules/resume-cli/build/render-html.js
+++ b/node_modules/resume-cli/build/render-html.js
@@ -21,10 +21,10 @@ var _default = async ({
   let path;
 
   if (themePath[0] === '.') {
-    path = tryResolve(path.join(cwd, themePath), {
+    path = tryResolve(require('path').join(cwd, themePath), {
       paths: [cwd]
     });
-    throw new Error(`Theme ${themePath} could not be resolved relative to ${cwd}`);
+    if(!path) throw new Error(`Theme ${themePath} could not be resolved relative to ${cwd}`);
   }
 
   if (!path) {
